skyportal/handlers/api/public_pages/public_source_page.py

```diff
@@ -80,6 +81,17 @@ def get_photometry(source_id, group_ids, stream_ids, session):
     return [photo.to_dict_public() for photo in session.scalars(stmt).unique().all()]
 
 
+def get_phot_stats(source_id, phot_stats, session):
+    if phot_stats is not None:
+        return phot_stats[0]
+
+    return session.scalar(
```

Comment:
could you just wrap that in a try except or something? The query could return None if the source has no phot stats yet. Unlikely but could be a point of failure that we want to avoid.